paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -2181,6 +2208,18 @@ public PartitionExpireStrategy partitionExpireStrategy() {
         return options.get(PARTITION_EXPIRATION_STRATEGY);
     }
 
+    public String dataFileExternalPaths() {
```

Comment:
Please add nullable for all nullable methods and fields. ALL, not just this one.